File: timeConversion/timeConversion.js
const time = '07:05:45PM'
const timeTwo = '02:05:20AM'
const timeThree = '12:03:20AM'
const timeFour = '10:05:15PM'
const timeFive = '11:15:30AM'

function timeConversion(time) {
    let day = time.slice(8,9)
    let hours = time.slice(0,2)
    let minutes = time.slice(3,5)
    let seconds = time.slice(6,8)
    let twelve = time.slice(0,2)
    let timeReturn = ''

    if (twelve === '12') {
        hours = '00'
        timeReturn = `${hours}:${minutes}:${seconds}`
    }

    if (day === 'P'){
        hours = parseInt(hours) + 12
        timeReturn = `${hours}:${minutes}:${seconds}`
    }

    if (day === 'A'){
        timeReturn = `${hours}:${minutes}:${seconds}`
    }

    console.log(timeReturn);
    return timeReturn
}


timeConversion(time)
timeConversion(timeTwo)
timeConversion(timeThree)
timeConversion(timeFour)
timeConversion(timeFive)

File: timeConversion/time_conversion.py
import math
import os
import random
import re
import sys

time = '07:05:45PM'
timeTwo = '02:05:20AM'
timeThree = '12:03:20AM'
timeFour = '10:05:15PM'
timeFive = '11:15:30AM'

def timeConversion(s):
    day = s[8:9]
    hours = s[0:2]
    minutes = s[3:5]
    seconds = s[6:8]
    time_return = ''
    
    if hours == '12':
        hours = '00'
        time_return = hours + ':' + minutes + ':' + seconds

    if day == 'A':
        time_return = hours + ":" + minutes + ':' + seconds

    if day == 'P':
        hours = int(hours) + 12
        time_return = str(hours) + ':' + minutes + ':' + seconds
    
    print(time_return)
    return(time_return)




timeConversion(time)
timeConversion(timeTwo)
timeConversion(timeThree)
timeConversion(timeFour)
timeConversion(timeFive)
